src/handler.py

```diff
@@ -214,12 +214,37 @@ def troubleshoot(event, context):
         _make_kms_key(event)
     elif event['action'].lower() == 'change_secret_kms_key':
         _change_secret_kms_key(event)
+    # TODO remove
+    elif event['action'].lower() == 'delete_stack':
+        client = boto3.client('cloudformation', "us-west-2")
+        response = client.delete_stack(
+            StackName='WQP-GEOSERVER-ECS-SERVICE-TEST',
+        )
+    elif event['action'].lower() == 'create_access_point':
+        _make_efs_access_point(event)
+    elif event['action'].lower() == 'create_fargate_security_group':
+        _make_fargate_security_group(event)
+    elif event['action'].lower() == 'delete_fargate_security_group':
+        # When you need to delete a security group, modify the code
+        # here and specify the group id.  Don't check into master
+        client = boto3.client('ec2', os.getenv('AWS_DEPLOYMENT_REGION'))
+        client.delete_security_group(GroupId='sg-xxxxxxxxxxxxxxxxx')
+    elif event['action'].lower() == 'delete_access_point':
+        # When you need to delete an efs access point, modify the code
+        # here and specify the access point id.  Don't check into master
```

Comment:
Just wanted to point out your comments to not check into master in case they're still relevant.

Reply:
I was trying to say "don't add a real id and then check that into master."  I think xxxxx is okay because it shows the developer what to do.  I didn't want to pass the id in the event, because I didn't mean for the troubleshoot lambda to become a way to willy-nilly do anything.  Like delete_stack(event['stack_name']) .... no ...

Reply:
but there has to be a way to construct these weird resources that are potentially shared across services and are set and forget otherwise.